public/docs/ts/latest/guide/reactive-forms.jade

```diff
@@ -0,0 +1,879 @@
+  At the moment, you're dumping the entire form model onto the page.
+  Sometimes you're interested only in the state of one particular `FormControl`.
+
+  You can inspect an individual `FormControl` within a form by extracting it with the `.get()` method.
+  Normally you'd do this _within_ the component class. 
+  But you can display it on the page by adding the following to the template, 
+  immediately after the `{{form.value | json}}` interpolation. 
+
++makeExample('reactive-forms/ts/app/hero-detail-4.component.html', 'inspect-value','app/hero-detail.component.html')(format=".")
+
+:marked
+  To get the state of a `FormControl` that’s inside a `FormGroup`, use dot notation to path to the control.
+
++makeExample('reactive-forms/ts/app/hero-detail-4.component.html', 'inspect-child-control','app/hero-detail.component.html')(format=".")
+
+:marked
+  You can use this technique to display _any_ property of a `FormControl` 
+  such as one of the following:
+style td, th {vertical-align: top}
+table(width="100%")
+  col(width="10%")
+  col(width="90%")
+  tr
+    th Property
+    th Description
+  tr
+    td <code>myControl.value</code>
+    td
+      :marked
+         the value of a `FormControl`.
+  tr
+    td <code>myControl.status</code>
+    td
+      :marked
+        whether the `FormControl` is `valid`, 
+         `invalid`, `pending`, or `disabled`.
+  tr
+    td <code>myControl.pristine</code>
+    td
+      :marked
+         `true` if the user has _not_ changed the value in the UI.
+  tr
+    td <code>myControl.untouched</code>
+    td
+      :marked
+         `true` if the control user has not yet entered the HTML control
+          and triggered its blur event.
+                                  
+:marked
+  Learn about other `FormControl` properties in the 
+  [_AbstractControl_](../api/forms/index/AbstractControl-class.html) API reference.
+
+  One common reason for inspecting `FormControl` properties is to 
+  make sure the user entered valid values. 
+  Read about validating Angular forms in the 
+  [Form Validation](../cookbook/form-validation.html) guide. 
+
+.l-main-section
+a#data-model-form-model
+:marked
+  ## The _data model_ and the _form model_
+
+  At the moment, the form is displaying empty values.
+  The `HeroDetailComponent` should display values of a hero,
+  possibly a hero retrieved from a remote server.
+
+  The hero from the server is the **_data model_**.
+  The `FormControl` structure is the **_form model_**.
+ 
+  The component must copy the hero values in the _data model_ into the _form model_.
+  There are two important implications:
+
+  1. The developer must understand how the properties of the _data model_ 
+  map to the properties of the _form model_.
+
+  2. User changes flow from the DOM elements to the _form model_, not to the _data model_.
+  The form controls never update the _data model_.
+  In the reactive paradigm, the data model is presumed to be _immutable_
```

Comment:
Yes ... really want @kara perspective on this.  She has been so adamant about immutability in the past ... and that's the only argument against having `NgModel` in `ReactiveFormsModule` AFAIK.